test/include/test_runner/test_group.h

```diff
@@ -18,6 +20,9 @@ struct TestStatement {
     bool enumerate = false;
     bool checkOutputOrder = false;
     std::string expectedTuplesCSVFile;
+    // for multiple conns
+    std::optional<std::string> conn_name; // default conntion name conn0
+    bool reLoadDBFlag = false;
```

Comment:
reLoadDBFlag -> reloadDBFlag

Reply:
@hououou Ditto. I don't see this comment addressed.